xml/ha_storage_protection.xml

```diff
@@ -920,6 +920,16 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
     the quorum and some reasonable assumptions. No &stonith; SBD resource
     primitive is needed in the CIB.
    </para>
+   <important>
+    <title>Do not block Corosync traffic in the local firewall</title>
+    <para>
+     Diskless SBD relies on reformed membership and loss of quorum to achieve
+     fencing. Corosync traffic must be able pass through all network interfaces,
```

Comment:
Sorry, not a native speaker here, but should it be "be able to ..."?

Reply:
Thank you, you're right. I must have read it so many times that I didn't notice the missing "to". Fixing now.

Reply:
Thanks @gao-yan and @tahliar looks good to me. And initially I've misspelled in the comment.